modules/monitoring-accessing-the-alerting-ui.adoc

```diff
@@ -27,22 +27,39 @@
 :perspective: Developer
 // end::DEV[]
 
-The Alerting UI is accessible through the *{perspective}* perspective of the {ocp-product-title} web console.
-
 // tag::ADM[]
+You can access the *Alerting* user interface (UI) is through the *{perspective}* perspective of the {ocp-product-title} web console.
```

Comment:
"is through" - I think the "is" here needs to be removed

Reply:
fixed